Documented Twig Component attribute (#2793)
ibexa/documentation-developer: docs/templating/components.md

### PHP code

> Create a class implementing the `\Ibexa\Contracts\TwigComponents\ComponentInterface` interface.

Reviewer: ```suggestion
Create a class that implements the `\Ibexa\Contracts\TwigComponents\ComponentInterface` interface.
```

> Create a class implementing the `\Ibexa\Contracts\TwigComponents\ComponentInterface` interface. 

Author: ```suggestion
Create a class implementing the `\Ibexa\Contracts\TwigComponents\ComponentInterface` interface.
```

Create a class implementing the `\Ibexa\Contracts\TwigComponents\ComponentInterface` interface and register it as a service by using the `ibexa.twig.component` service tag, for example:

=== "PHP Attribute2"

    ``` php

>         [[= include_file('code_samples/back_office/components/MyComponent.php') =]]

Reviewer 2: ```suggestion
    [[= include_file('code_samples/back_office/components/MyComponent.php', 0, None, '    ') =]]
```

Author: Thanks! I've decided to use python's named arguments to improve it, but wouldn't be able to do so without your help 🙇
